app/src/test/kotlin/com/mapzen/erasermap/presenter/MainPresenterTest.kt

```diff
@@ -374,6 +374,13 @@ public class MainPresenterTest {
         assertThat(mainController.routeLine).isEqualTo(route)
     }
 
+    @Test fun onMuteClick_shouldToggleMute() {
+        val muted = mainController.muted
+        presenter.onMuteClick()
+        val mutedAfter = mainController.muted
+        assertThat(muted != mutedAfter)
```

Comment:
Fluent assertion syntax requires a actual, expected, and a condition predicate to work correctly.

```kotlin
assertThat(actual).condition(expected)
```

In this case you would want to replace what you have with:

```kotlin
assertThat(mutedAfter).isNotEqualTo(muted)
```